packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -45,39 +49,38 @@ export const TRANSLATED_INPUT_TYPES = Object.keys(
 export type TranslatedInputType = (typeof TRANSLATED_INPUT_TYPES)[number];
 
 /**
- * Translated error messages that inform about causes for invalidity of form components
+ * Translated messages that inform about the validity state of form components
  */
-export type FormErrorMessages = {
+export type FormMessages = {
   /**
-   * A short error message preview to inform the user about the cause of the error
+   * A short message preview to inform the user about the validity state
    */
   shortMessage: string;
   /**
-   * An extended informative error message to provide more info
-   * how the error cause can be resolved
+   * An extended informative message to provide more info
    */
   longMessage?: string;
 };
 
 /**
- * Transforms a customError into the format needed to display an error preview and extended message
+ * Transforms a customMessage into the format needed to display an error preview and extended message
  */
-export const getCustomErrors = (customError?: CustomErrorType): FormErrorMessages | undefined => {
-  if (!customError) return;
-  if (typeof customError === "string") {
-    // we can't guarantee a custom error message will be short,
+export const getCustomMessages = (customMessage?: CustomMessageType): FormMessages | undefined => {
+  if (!customMessage) return;
+  if (typeof customMessage === "string") {
+    // we can't guarantee a custom message will be short,
     // so in case it overflows, by adding it to "longMessage",
     // it will still be visible in a tooltip
-    return { shortMessage: customError, longMessage: customError };
+    return { shortMessage: customMessage, longMessage: customMessage };
   }
-  return customError;
+  return customMessage;
 };
 
 /**
- * Returns a string combining short + long message or just the customError if it was provided as single string.
+ * Returns a string combining short + long message or just the customMessage if it was provided as single string.
  * Will be used e.g. for customInvalidity and showing a tooltip e.g. in RadioButtons
  */
-export const getCustomErrorText = (customError?: CustomErrorType): string | undefined => {
+export const getCustomText = (customError?: CustomMessageType): string | undefined => {
```

Comment:
```suggestion
export const getCustomText = (customError?: CustomMessageType): string | undefined => {
```

What do you think about renaming this to e.g. `getFormMessageText`? So its more clear what it does
Similar, `getCustomMessages` could be renamed to `getFormMessages`